blog/_posts/2016-10-12-Hackbright.md

```diff
@@ -0,0 +1,123 @@
+---
```

Comment:
There's a slight issue with the indentation of all the content of the file and Jekyll doesn't seem to like that very much — could you go back into your editor and remove the space at the beginning of each line (aka "Unindent")? (If you use sublime, it's Select All then `cmd` + `[`, it should be similar in Atom or other editors)